tuf_conformance/test_keys.py

```diff
@@ -214,3 +214,48 @@ def test_keytype_and_scheme(
 
     assert client.refresh(init_data) == 1
     assert client.version(Root.type) == 2
+
+
+def test_invalid_sig_in_valid_metadata(
+    client: ClientRunner,
+    server: SimulatorServer,
+) -> None:
+    """Test that client accepts valid metadata if it contains an invalid signature
+
+    Throughout the test timestamp is signed by two keys: one signature is always correct
+    and that is enough to reach threshold. First part uses a plausible looking signature
```

Comment:
Not clear what "First part" and "second part" refer to.

Reply:
cheers, Tweaked docstring to refer to first and second refresh.